src/components/PathwayExercise/ExerciseContent/Assessment/styles.js

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles((theme) => ({
+  option: {
+    boxShadow:
+      "0px 1px 2px #48A145, 0px 2px 1px #48A145, 0px 1px 5px #48A145 !important",
```

Comment:
```suggestion
      `0px 1px 2px ${theme.palette.primary.main}, 0px 2px 1px ${theme.palette.primary.main}, 0px 1px 5px ${theme.palette.primary.main} !important`,
```

Reply:
![Screenshot from 2022-05-16 20-17-42](https://user-images.githubusercontent.com/67674096/168620424-e31caa46-0022-4675-876f-2b7f7c4e6dbd.png)

It's showing this error

Reply:
This is very likely because the `theme` is not being passed as an input to `makeStyles` (instead it's passing in `{}`).  Do you have a route to this Component yet?  I think this problem (https://stackoverflow.com/questions/69032871/react-material-ui-themeprovider-not-being-applied) might be resolved if it becomes a descendant of a `ThemeProvider`.